### DISCUSSION TITLE
HPR-nummer

### DISCUSSION THREAD

**Discussion by AMJohnsen** [NorskHelsenett/Tillitsrammeverk#3] [Q&A] [answered]
Hvorfor er det valgfritt å oppgi HPR-nummer? PID er obligatorisk. Er tanken å hente ut HPR gjennom PID? @steinarnoem

**Comment by steinarnoem** [comment, marked as answer]
I prinsippet skal alt helsepersonell med tjenstlig behov ha tilgang til relevante og nødvendige helseopplysninger, uavhengig av om de har formelle autorisasjoner og/eller lisenser. I tillegg kan administrativt personell, som f.eks. legesekretær, hente opplysninger på vegne av helsepersonell.
Samtidig kan det være behov for å styre og kontrollere tilgang til enkelte typer helseopplysninger - hvor tilgangen er basert på en sannsynligjøring av at helsepersonellet f.eks. er en lege eller psykolog. Derfor er det ansett som fornuftig å legge ved denne informasjonen sentralt dersom den finnes for å unngå at alle datakilder selv må gjøre oppslag mot HPR.

Høres det sånn ca riktig ut, @richardhus?

**Reply by richardhus** [reply]
Ja, det er det jeg også tenker at man må legge til grunn for fremtidige samhandlingsløsninger. Legesekretærer uten formell autorisasjon som helsepersonell kan for eksempel ha behov for å se (og redigere?) timebok i andre virksomheter for pasientene som er/skal/har vært til behandling av virksomheten de jobber for. Tilsvarende for andre mer administrativt orienterte helseopplysninger som de trenger for å sikre helhetlige og gode pasientforløp logistikkmessig.

**Reply by mortsten** [reply]
Jeg tenker den bør legges ved gitt at helsepersonellet har autorisasjon, men at det for andre grupper ikke er påkrevd.

**Comment by steinarnoem** [comment]
Behandlet av feature team, som konkluderte med at autorisasjoner og lisenser ikke bør inngå i informasjonsmodellen.

Denne tråden låses.

Ref:
https://github.com/NorskHelsenett/Tillitsrammeverk/blob/main/meeting_minutes/2023-03-07%20temam%C3%B8te.md
https://github.com/NorskHelsenett/Tillitsrammeverk/issues/42
